build.sbt

```diff
@@ -56,7 +58,7 @@ resolvers ++= Seq(
 )
 
 buildInfoPackage := "app"
-buildInfoKeys += "gitCommitId" -> env.getOrElse("GITHUB_SHA", "Unknown")
+buildInfoKeys += "gitCommitId" -> BuildInfo(baseDirectory.value).revision
```

Comment:
That looks like the right value to target - should it have a fallback as well, or maybe be a lazy val so it only gets computed when a build is being made for prod? That seems to be happening [elsewhere](https://github.com/guardian/frontend/blob/6359c5eeda42e9ef0a611807208650df409229a6/project/ProjectSettings.scala#L142).

Reply:
Yep, the logic we're calling here [does do that fallback](https://github.com/guardian/sbt-riffraff-artifact/blob/de3189974368d3b607a57360491ab38331e41a16/src/main/scala/com/gu/riffraff/artifact/BuildInfo.scala#L163-L168) already, so we don't need to add our own fallback code!

Reply:
Makes sense - I'd seen where `revision` was getting set but didn't spot the `unknown` part! Sounds good to me then 👍